dali/pipeline/operator/builtin/external_source.h

```diff
@@ -415,9 +494,10 @@ class ExternalSource : public Operator<Backend> {
    */
   struct ExternalSourceState {
     bool copied_shared_data = false;
+    size_t batch_size = 0;
```

Comment:
I see only modifications putting the current bs in, but no access to it?

Reply:
Removed